yellowstone-grpc-geyser/config.json

```diff
@@ -5,13 +5,13 @@
   },
   "grpc": {
     "address": "0.0.0.0:10000",
-    "tls_config": {
```

Comment:
Remove this tls config from the sample config because it's not a valid tls config.